util/grpc/src/lib.rs

```diff
@@ -308,6 +308,8 @@ pub fn rpc_logger(ctx: &RpcContext, logger: &Logger) -> Logger {
 }
 
 lazy_static::lazy_static! {
+    /// Generates service metrics for tracking
+    pub static ref SVC_COUNTERS: ServiceMetrics = ServiceMetrics::new_and_registered("grpc");
```

Comment:
I see the screen shot shows metrics from the AdminAPI, is this used across projects for grpc services/routes? 

Ideally this would be prefixed with the service name and not just `grpc`.  Something like `consensus_grpc` or `fog_ledger_grpc`.  Is there a way to pass the service name in as a parameter when these metrics get registered?

In any case this is better that just  `duration_bucket`  :+1:

Reply:
I'll check if I can pass the service name as a param as Its a bigger change if it has to be `consensus_grpc` etc., as its used across the project.